### ISSUE TITLE
Example config lacks required [logging] field

### ISSUE THREAD

**Issue by zu3st** [cea-sec/openwec#112]
On following the Getting-Started Guide, storing the given config and calling `openwec db init` as instructed fails with the following error:
https://github.com/cea-sec/openwec/blob/38161114cc113ea9a8543469abdc063614153337/doc/getting_started.md?plain=1#L34-L51

```
An error occurred: Failed to retrieve configuration: TOML parse error at line 1, column 1
  |
1 | # /etc/openwec.conf.toml
  | ^
missing field `logging`
```
Adding the line 
```
[logging]
```
to the config resolves the issue.

**Comment by vruello** [comment]
Hi! Nice catch :smile: 

This has been fixed in #108 (https://github.com/cea-sec/openwec/pull/108/commits/6b0d19933c05e87b712e3c007365c5b8c4a6828b) which should be merged very soon.

**Comment by vruello** [comment]
#108 has been merged.
